deps/llhttp/src/api.c

```diff
@@ -53,7 +54,7 @@ int on_headers_complete(llhttp_t * p)
 }
 
 const llhttp_settings_t wasm_settings = {
-  NULL,
+  wasm_on_message_begin,
```

Comment:
@mcollina previously we didn't use this llhttp callback and just passed NULL.

Reply:
I mean.. this change shouldn't happen upstream?

Reply:
We have other changes here that are not upstream.

Reply:
I'm not sure it's worth trying to port these changes upstream as it's just C glue between undici and llhttp.

Reply:
We should move them upstream as it will be simpler to bump the library.

Reply:
https://github.com/nodejs/llhttp/pull/116